### PR TITLE
Added Getting Started Sample For Navigation Drawer, Segment, Effect View Control

### PR DESCRIPTION
**Pull request by Sathish-087**
**Description**
Added getting started sample for navigation drawer, segment, and effect view control.

**Output Screenshots**

- Navigation Drawer

https://github.com/user-attachments/assets/dff148a6-ee80-4f37-8860-e3f7fe754bc8

- Segment

image

- EffectsView

https://github.com/user-attachments/assets/1dbca38d-1bc6-4205-897e-3e997fda8f62

### CODE REVIEW

File: NavigationDrawer/GettingStarted/GettingStarted/MainPage.xaml.cs [hunk 1]
```diff
@@ -0,0 +1,43 @@
+﻿using Microsoft.Maui.Platform;
+using Syncfusion.Maui.Toolkit.NavigationDrawer;
+
+namespace GettingStarted
+{
+    public partial class MainPage : ContentPage
+    {
+        public MainPage()
+        {
+            InitializeComponent();
```
Comment (MuniappanSubramanian) [inline NavigationDrawer/GettingStarted/GettingStarted/MainPage.xaml.cs:10]: move this changes in behavior

Comment (Sathish-087) [reply NavigationDrawer/GettingStarted/GettingStarted/MainPage.xaml.cs:7]: Changed to behavior

File: NavigationDrawer/GettingStarted/GettingStarted/Behavior/GettingStartedBehavior.cs [hunk 2]
```diff
@@ -0,0 +1,144 @@
...
+
+        /// <summary>
+        /// Called when the behavior is attached to a content page.
+        /// </summary>
+        /// <param name="bindable">The content page.</param>
+        protected override void OnAttachedTo(ContentPage bindable)
+        {
+            base.OnAttachedTo(bindable);
+            bindable.Appearing += Bindable_Appearing;
+        }
```
Comment (MuniappanSubramanian) [inline NavigationDrawer/GettingStarted/GettingStarted/Behavior/GettingStartedBehavior.cs:43]: Appearing needed?

Comment (Sathish-087) [reply NavigationDrawer/GettingStarted/GettingStarted/Behavior/GettingStartedBehavior.cs]: Checked and removed this method

**Review by Jeyasri-Murugan — APPROVED** [review]

**Review by Jeyasri-Murugan — APPROVED** [review]

**Pull request merged**